.github/workflows/verify_react.yml

```diff
@@ -35,7 +35,7 @@ jobs:
           cache-dependency-path: ./frontend/package-lock.json
 
       - name: Install our dependencies
-        run: npm ci --force
```

Comment:
what does it do?

Reply:
[npm ci](https://docs.npmjs.com/cli/v8/commands/npm-ci) installs denpencies from the `package-lock.json` file, where all dependency versions are set. There were some conflicts between dependencies before I cleaned the dependencies. Now, there are no conflicts so we don't need to `--force` the installation.